app/models/email_log.rb

```diff
@@ -87,6 +83,10 @@ def bounce_key
     super&.delete('-')
   end
 
+  def topic
```

Comment:
I think it is fine to keep this method but I don't think we need to change the commit history here.

Reply:
Oh I must have put it in the wrong place sorry